iphone/Maps/UI/PlacePage/PlacePageViewController.swift

```diff
@@ -104,6 +78,7 @@ final class PlacePageScrollView: UIScrollView {
     // Update layout when the device was rotated but skip when the appearance was changed.
     if self.previousTraitCollection != nil, previousTraitCollection?.userInterfaceStyle == traitCollection.userInterfaceStyle {
       DispatchQueue.main.async {
+        self.beginDragging = false
```

Comment:
Why is it needed? A comment may be helpful.

Reply:
This one is removed because isnt needed! Thanks!